### PR TITLE
Create header element and styles

### PR DESCRIPTION
**Pull request by maxnixon**
[Issue #3](https://github.com/robcapell/maxnixon-practice-shoe_mockup/issues/3)

### CODE REVIEW

File: productDetailPage.html [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
     <link href="css/productPage/productBaseStyles.css" rel="stylesheet">
     <link href="css/productPage/productModuleStyles.css" rel="stylesheet">
     <link href="css/productPage/productLayoutStyles.css" rel="stylesheet">
     <link href="css/productPage/productStateStyles.css" rel="stylesheet">
     <link href="css/productPage/productThemeStyles.css" rel="stylesheet">
+    <link href="css/reset.css" rel="stylesheet">
   </head>
   <body>
+
+    <div id="header-container">
```
Comment (robcapell) [inline productDetailPage.html]: Why is this not an html5 `<header>` tag?

File: productDetailPage.html [hunk 2]
```diff
@@ -1,16 +1,40 @@
...
+          <img id="addidasLogo" src="home/mnixon/Sites/maxnixon-practice-shoe-mockup/images/addidasLogo.jpg" alt="addidas logo">
+          <div id="topNavBar">
+            <nav class="white">Track Your Order</nav>
```
Comment (robcapell) [inline productDetailPage.html]: Incorrect use of the `<nav>` tag. The `<nav>` tag should as as a container tag for your websites navigation or sub navigation. Its purpose is to declare a group of links as specific to website navigation.
I also see no `<ul>` tags. A websites navigation is a list. Make sure to use html correctly.
Your nav items also are not in `<a>` tags. Make sure to use them, and as a dummy placeholder, you can set href to equal "#".

File: productDetailPage.html [hunk 3]
```diff
@@ -1,16 +1,40 @@
...
     <link href="css/productPage/productModuleStyles.css" rel="stylesheet">
     <link href="css/productPage/productLayoutStyles.css" rel="stylesheet">
     <link href="css/productPage/productStateStyles.css" rel="stylesheet">
     <link href="css/productPage/productThemeStyles.css" rel="stylesheet">
+    <link href="css/reset.css" rel="stylesheet">
   </head>
   <body>
+
+    <div id="header-container">
+          <img id="addidasLogo" src="home/mnixon/Sites/maxnixon-practice-shoe-mockup/images/addidasLogo.jpg" alt="addidas logo">
```
Comment (robcapell) [inline productDetailPage.html]: This is an absolute link!
Nice `<alt>` tag! lets make sure we are also adding `<title>` tags.

File: productDetailPage.html [hunk 4]
```diff
@@ -1,16 +1,40 @@
...
+          </div>
+          
+          <div id="bottomNavBar">
+            <nav>MEN</nav>
+            <nav>WOMEN</nav>
+            <nav>KIDS</nav>
+            <nav>SPORTS</nav>
+            <nav>BRANDS</nav>
+            <nav>CUSTOMIZE</nav>
+            <nav><input type="search" placeholder="search"></nav>
```
Comment (robcapell) [inline productDetailPage.html]: Nice usage of placeholder text!

File: css/productPage/productLayoutStyles.css [hunk 5]
```diff
@@ -0,0 +1,43 @@
+#header-conainer {
```
Comment (robcapell) [inline css/productPage/productLayoutStyles.css]: After you make changes to the header, and use HTML5 tags for the headers markup. Change your selectors to reference the tag names themselves, vs having an ID as this is uneeded. For example `header {...}`.

**Comment by robcapell** [comment]
Remember to keep in mind the writing your styles first approach, and thinking of classes as reusable  modular components that can be mixed and matched to manipulate the look of your html.

File: productDetailPage.html (hunk 3 above)
Comment (maxnixon) [inline productDetailPage.html]: title tags... as in inside the img tag Or just in general

**Pull request closed without merging**